modules/upgrade/pages/client-migration.adoc

```diff
@@ -157,3 +157,36 @@ If the system status reports that the migration has failed, but also shows that
 After the migration is complete, schedule a reboot of the client from the {productname} Server.
 
 ////
+
+
+
+== SP Mass Migration
+
+If you want to migrate many clients to the next SP version, {suse} recommends to use {productname} API calls.
+
+You need to know to which migration target you want to migrate.
```

Comment:
I think this should probably be a procedure.

Reply:
ok, trying to do my best ;)